code/_onclick/item_attack.dm

```diff
@@ -12,6 +12,18 @@
 		return FALSE
 	return target.tool_act(user, src, tool_behaviour)
 
+
+/obj/item/proc/surgery_attack_chain(mob/user, atom/target)
```

Comment:
Given this is `mob/living` level stuff you might want to reflect that in these args

Reply:
Good point. Will change